### PR TITLE
chore(GAE flex): rm third-party email samples

### PR DESCRIPTION
**Pull request by ace-n**
@qchong please LGTM too, thanks!

(FYI @fhinkel: please wait until everyone LGTMs before merging, unless you know for sure these samples are no longer used.)

### CODE REVIEW

**Comment by ace-n** [comment]
@fhinkel once this is approved, I'll go ahead and remove the appropriate test triggers internally.

**Comment by qchong** [comment]
LGTM

**Review by fhinkel — APPROVED** [review]

**Comment by ace-n** [comment]
Internal test triggers have been removed (cl/353662852). Will merge once tests pass.

**Pull request merged**
